packages/editor/src/components/post-publish-panel/postpublish-next-panel-template.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { useRegistry, useSelect } from '@wordpress/data';
+import { decodeEntities } from '@wordpress/html-entities';
+import { store as coreStore } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import { store as editorStore } from '../../store';
+
+function focusRef( node ) {
+	node?.focus();
+}
+
+export function PostPublishNextPanelTemplate( { post, focusOnMount } ) {
```

Comment:
Are we settled on a separate post publish for "template"? Can't we unify?

Reply:
What isn't unified? The "what's next?" panel? How can we unify that? There's no view post and "post address".